src/app/public/src/modules/edit-button/edit-button.component.html

```diff
@@ -1,10 +1,6 @@
-<div class="stache-btn-edit">
-  <button
-    *ngIf="url"
-    type="button"
-    class="sky-btn sky-btn-link"
-    (click)="onEditButtonClick()"
-  >
-    <i class="fa fa-edit"></i> {{ editButtonText }}
-  </button>
+<div class="stache-edit-button" *ngIf="url">
+  <a class="stache-btn-edit" [href]="url">
```

Comment:
Since the action is opening a URL, we should use an anchor, instead of a button.